.github/workflows/build.yml

```diff
@@ -102,20 +104,20 @@ jobs:
     needs: 'build'
     if: github.ref=='refs/heads/main' || github.ref=='refs/heads/dev'
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Set up Python 3.8
         uses: actions/setup-python@v2
         with:
           python-version: 3.8
-      - uses: actions/download-artifact@v2
+      - uses: actions/download-artifact@v4
         with:
           name: dist
           path: dist/
       - name: Generate SHA256 files for each wheel
         run: |
           sha256sum dist/*.whl > checksums.txt
```

Comment:
can you also have it do `sha256sum dist/*.tar.gz >> checksums.txt`? It lets us match that what we built on our github action runners actually matches what is on pypi.org on publication :)